aio/aio-proxy/aio_proxy/response/formatters/non_diffusible.py

```diff
@@ -7,4 +7,57 @@ def hide_non_diffusible_fields(result_formatted):
     for field in result_formatted.keys():
```

Comment:
Why use this for loop ? 

Cant you just write : 

```
for field in hidden_fields:
   result_formatted[field] = "[NON-DIFFUSIBLE]"

if result_formatted["siege"]:
   result_formatted["siege"] = hide_non_diffusible_etablissement_fields(
                result_formatted["siege"]
  )

if result_formatted["dirigeants"]:
   ...
```

Seems easier to read and saves a for loop.

Thats actually how you implemented it in `hide_non_diffusible_etablissement_fields`